model/huggingface/infer.py

```diff
@@ -1,22 +1,18 @@
 from dataclasses import dataclass
 from typing import Callable, Tuple, List
-from transformers import pipeline
+
 from config import HuggingfaceConfig, huggingface_config
 from datasets import Dataset
 import numpy as np
 from type import Label, Probabilities
-
-
-def load_module(module_name: str) -> Callable:
-    return pipeline(task="sentiment-analysis", model=module_name)
+from model.base import BaseModel
 
 
 def run_inference_pipeline(
-    test_data: Dataset, config: HuggingfaceConfig
+    model: BaseModel, test_data: Dataset, config: HuggingfaceConfig
```

Comment:
is this BaseModel? or a huggingface model already?

Reply:
Fixed it!